src/uwtools/templater.py

```diff
@@ -0,0 +1,109 @@
+#!/usr/bin/env python3
+
+'''
+This utility renders a Jinja2 template using user-supplied configuration options
+via YAML or environment variables.
+'''
+
+import os
+import sys
+import argparse
+
+from jinja2 import Environment, FileSystemLoader, meta
+import f90nml
+
+from uwtools import config
+
+def file_exists(arg):
```

Comment:
Nitpicky comment: `file_exists()` to me implies that a boolean would be returned and the function would just check if the file exists. Maybe `path_if_file_exists()` or something along those lines would be more descriptive.